case-study-azure-devops.md

```diff
@@ -0,0 +1,416 @@
+    <td>Limited source control and continuous integration features compared to other platforms. User interface can be complicated for some.</td>
+    <td>
+        <ul style="list-style: none; padding: 0; color: black;">
+            <li>
+                <a href="https://www.atlassian.com/software/jira" style="color: #ffffff; text-decoration: underline #ffffff;">Jira software</a>
+            </li>
+            <li>
+                <a href="https://sourceforge.net/software/product/JIRA/" style="color: #ffffff; text-decoration: underline #ffffff;">Sourceforge</a>
+            </li>
+        </ul>
+    </td>
+  </tr>
+  <tr style="background-color: #ffffff; color: black;">
+    <td>GitHub</td>
+    <td>Free for public repositories and starts at $4 per user/month for private repositories.</td>
+    <td>GitHub provides source control and version control, as well as project management and collaboration features. It also offers continuous integration and deployment, pull requests, and issues.</td>
+    <td>Widely used and supported by a large developer community. Good for open-source projects. Offers integrations with many other tools and services.</td>
+    <td>Limited project management and agile planning features compared to other platforms. User interface can be cluttered for some.</td>
+    <td>
+        <ul style="list-style: none; padding: 0; color: black;">
+            <li>
+                <a href="https://github.com/about" style="color: #131313; text-decoration: underline #131313;">GitHub</a>
+            </li>
+            <li>
+                <a href="https://sourceforge.net/software/product/GitHub/" style="color: #131313; text-decoration: underline #131313;">Sourceforge</a>
+            </li>
+        </ul>
+    </td>
+  </tr>
+  <tr style="background-color: #3E3E3E; color: white;">
+    <td>GitLab</td>
+    <td>Free for public repositories and starts at $4 per user/month for private repositories.</td>
+    <td>GitLab provides source control, continuous integration and deployment, security, and project management. It also offers issue tracking, wikis, and dashboards.</td>
+    <td>Good for DevOps and continuous delivery. Offers a comprehensive solution for software development. Strong focus on security and compliance.</td>
+    <td>User interface can be complicated for some. Not as widely used or supported as other platforms.</td>
+    <td>
+        <ul style="list-style: none; padding: 0; color: black;">
+            <li>
+                <a href="https://about.gitlab.com/" style="color: #ffffff; text-decoration: underline #ffffff;">GitLab</a>
+            </li>
+            <li>
+                <a href="https://sourceforge.net/software/product/GitLab/" style="color: #ffffff; text-decoration: underline #ffffff;">Sourceforge</a>
+            </li>
+        </ul>
+    </td>
+    </tr>
+</table>
+
+To create comparison between the software above i have used [this website](https://sourceforge.net/)
+
+## **How to**
```

Comment:
Not finished this section yet?

Reply:
Not yet. It will come a video here on how to create a account, add board etc. 
Not sure how much text is needed here to showcase how to use it as the CA says.